tidb-cloud/import-parquet-files.md

```diff
@@ -83,71 +83,71 @@ Because Parquet files do not contain schema information, before importing data f
+        - **File Name**: enter a pattern that matches the names of the Parquet files to be imported. If you have one Parquet file only, you can enter the filename here directly.
 
             For example:
 
             - `my-data?.parquet`: all Parquet files starting with `my-data` and one character (such as `my-data1.parquet` and `my-data2.parquet`) will be imported into the same target table.
             - `my-data*.parquet`: all Parquet files starting with `my-data` will be imported into the same target table.
 
-        - **Target Table Name**: enter the name of the target table in TiDB Cloud, which must be in the `${db_name}.${table_name}` format. For example, `mydb.mytable`. Note that this field only accepts one specific table name, so wildcards are not supported.
+        - **Target table**: enter the name of the target table in TiDB Cloud, which must be in the `${db_name}.${table_name}` format. For example, `mydb.mytable`. Note that this field only accepts one specific table name, so wildcards are not supported.
 
-4. Click **Import**.
+    - **Tables Filter**: if you want to filter which tables to be imported, you can specify one or more table filters in this field, separated by `,`.
+
+        For example:
+
+        - `db01.*`: all tables in the `db01` database will be imported.
+        - `db01.table01*,db01.table02*`: all tables starting with `table01` and `table02` in the `db01` database will be imported.
+        - `!db02.*`: except the tables in the `db02` database, all other tables will be imported. `!` is used to exclude tables that do not need to be imported.
+        - `*.*` : all tables will be imported.
+
+        For more information, see [table filter snytax](/table-filter.md#syntax).
 
-    A warning message about the database resource consumption is displayed.
+5. Click **Next**.
 
-5. Click **Confirm**.
+6. On the **Preview** page, confirm the import data and then click **Start Import**.
 
     TiDB Cloud starts validating whether it can access your data in the specified bucket URL. After the validation is completed and successful, the import task starts automatically. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).
 
-6. When the import progress shows success, check the number after **Total Files:**.
+7. When the import progress shows **Completed**, check the number after **Total Files:**.
 
-    If the number is zero, it means no data files matched the value you entered in the **Object Name Pattern** field. In this case, check whether there are any typos in the **Object Name Pattern** field and try again.
+    If the number is zero, it means no data files matched the value you entered in the **File name** field. In this case, check whether there are any typos in the **File name** field and try again.
```

Comment:
"If the number is zero" should be "If the size is zero which is displayed on the  Summary board"